### PR TITLE
Date error with MDT CGM

### PR DESCRIPTION
**Pull request by cjo20**
Fixes the date/jq errors towards the end of the loop in MDT CGM. Needs testing by someone with non-MDT CGM before merging.

### CODE REVIEW

**Comment by luiesa** [comment]
fixed fine!

**Comment by cjo20** [comment]
Exactly what error are you getting?

File: bin/oref0-pump-loop.sh [hunk 1]
```diff
@@ -690,7 +690,11 @@ function wait_for_bg {
 
 function glucose-fresh {
     # set mtime of monitor/glucose.json to the time of its most recent glucose value
-    touch -d "$(date -R -d @$(jq .[0].date/1000 monitor/glucose.json))" monitor/glucose.json 2>&3
+    if grep "MDT cgm" openaps.ini 2>&3 >&4; then
```
Comment (scottleibrand) [inline bin/oref0-pump-loop.sh]: Would it be better to check for the existence of `.[0].display_time` here?  FWIW, my data from Dexcom includes both fields.

Comment (cjo20) [reply bin/oref0-pump-loop.sh]: This seems to be the check used to work out if it's MDT data or not elsewhere in the script, I was trying to be consistent. Do you have any preference?

Comment (scottleibrand) [reply bin/oref0-pump-loop.sh]: My Dexcom data has a display_time field as well, so it's not really a MDT-specific thing.  I think we should be able to just use display_time across the board, but probably safer for backwards compatibility to fall back to date if display_time doesn't exist?

Comment (cjo20) [reply bin/oref0-pump-loop.sh]: Ok, I'll try to take a look at it tomorrow.

**Pull request merged**